rest-api-spec/src/main/resources/rest-api-spec/test/indices.put_alias/all_path_options.yml

```diff
@@ -45,9 +45,9 @@ setup:
       indices.get_alias:
         name: alias
 
-  - match: {test_index1.aliases.alias: {}}
-  - match: {test_index2.aliases.alias: {}}
-  - match: {foo.aliases.alias: {}}
+  - match: {test_index1.aliases.alias: { is_write_index: true }}
+  - match: {test_index2.aliases.alias: { is_write_index: false }}
+  - match: {foo.aliases.alias: { is_write_index: false }}
 
 ---
 "put alias in * index":
```

Comment:
@bleskes, we discussed early about how it felt awkward to allow the defaulting of `is_write_index -> true` when 
not specified in alias updates apis (in addition to index creation). Towards the end of our discussion it seemed 
reasonable to rethink this and maybe enable this behavior. Although ordering was clear in the index-aliases api where actions are specified in a deterministic order, `indices.put_alias` allows resolution to multiple indices in such a way that ordering is undefined.

For these reasons, do you think it would be OK if we only default (when possible) `is_write_index -> true` during index creation?